operator/operator.yaml

```diff
@@ -16,6 +16,7 @@ tasks:
       - cassandra-env-sh.yaml
       - jvm-options.yaml
       - stateful-set.yaml
+      - cassandra-exporter-config-yml.yaml
```

Comment:
can we remove `-yml`? Seems redundant to say yaml yaml

Reply:
I just followed what been used already like "cassandra-yaml.yaml"
so first part is actual name followed by .yaml